app/src/main/res/layout/activity_camera.xml

```diff
@@ -58,4 +58,5 @@
             android:textColor="@android:color/black"
             tools:text="cat: 0.21423423" />
     </LinearLayout>
+
```

Comment:
empty line

Reply:
Removed...even though I prefer the newline here. :-)